bin/ch/WScriptJsrt.cpp

```diff
@@ -47,6 +47,7 @@
 MessageQueue* WScriptJsrt::messageQueue = nullptr;
 std::map<std::string, JsModuleRecord>  WScriptJsrt::moduleRecordMap;
 std::map<JsModuleRecord, std::string>  WScriptJsrt::moduleDirMap;
+std::map<JsModuleRecord, int>  WScriptJsrt::moduleErrMap;
```

Comment:
Can we replace int here with an enum? I'd like to avoid the use of magic numbers.